Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -2359,15 +2398,33 @@ def create_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> Comman
     if "summary" not in issue_fields.get("fields", {}):
         raise DemistoException("The summary argument must be provided.")
     res = client.create_issue(json_data=issue_fields)
-    outputs = {"Id": res.get("id", ""), "Key": res.get("key", "")}
-    markdown_dict = outputs | {"Ticket Link": res.get("self", ""), "Project Key": res.get("key", "").split("-")[0]}
-    return CommandResults(
+
+    ticket_url = res.get("self")
+    ticket_id = res.get("id") or res.get("ticket_id")
+    ticket_key = res.get("key", "")
+    formatted_ticket_id = f"{ticket_key}-{ticket_id}"
+    mirror_obj = MirrorObject(ticket_url=ticket_url, ticket_id=formatted_ticket_id)
+
+    outputs = {"Id": res.get("id", ""), "Key": ticket_key}
+    markdown_dict = outputs | {"Ticket Link": res.get("self", ""), "Project Key": ticket_key.split("-")[0]}
+    ticket_results = CommandResults(
         outputs_prefix="Ticket",
         outputs=outputs,
         outputs_key_field="Id",
         readable_output=tableToMarkdown(name=f'Issue {outputs.get("Key", "")}', t=markdown_dict),
-        raw_response=res,
+        raw_response=res
     )
+    results.append(ticket_results)
+
+    if demisto.command() == "jira-create-ticket-quick-action":
```

Comment:
I Prefer to pass a new flag to the function "create_issue_command" like "create_mirror_object) and only if it is true create the mirroring object and add it.
This is better than checking inside which command is it.